tests/PHPStan/Rules/PhpDoc/data/invalid-phpstan-doc.php

```diff
@@ -14,10 +14,10 @@ class Boo extends Baz
 	function foo(string $a){}
 
 	/**
-	 * @phpstan-ignore-next-line
```

Comment:
If you want, I can make this rule to use proper lines too and this remains working.

Reply:
Yes, please update this rule the same way the others, it's also about errors in PHPDocs.